src/resources/js/commerce/gateway/paypal/checkout.js

```diff
@@ -358,9 +374,13 @@ tribe.tickets.commerce.gateway.paypal.checkout = {};
 	 * @return {void}
 	 */
 	obj.ready = function () {
-		$document.on( 'afterSetup.tribeTicketsCommerceCheckout', obj.setupButtons );
+		obj.setupLoader();
+
+		window.onload = (event) => {
+			obj.setupButtons( event, $( tribe.tickets.commerce.selectors.checkoutContainer ) );
+		};
```

Comment:
Should this check whether the `checkoutContainer` exists before running any of this code? Saves us from running code that doesn't need to be run in the admin.

Reply:
the `tribe.tickets.commerce` JS is a dependency for this script. So, that gets loaded before this `ready` is fired.

We are only using this script in FE, so I am not sure where it can run in admin.